libs/template/chunk/src/lib/chunk.directive.ts

```diff
@@ -0,0 +1,178 @@
+import {
+  Directive,
+  inject,
+  Input,
+  NgZone,
+  OnDestroy,
+  OnInit,
+  TemplateRef,
+  ViewContainerRef,
+} from '@angular/core';
+import { RxStrategyProvider } from '@rx-angular/cdk/render-strategies';
+import { NextObserver, Subscription } from 'rxjs';
+
+/**
+ * @Directive ChunkDirective
+ *
+ * @description
+ *
+ * The `*rxChunk` directive serves as a convenient way for dividing template work into
+ * chunks. Applied to an element, it will schedule a task with the given RxRenderStrategy
+ * in order to postpone the template creation of this element.
+ *
+ * ### Features of `*rxChunk`
+ *
+ * - lightweight alternative to `*rxLet="[]"`
+ * - renderCallback
+ * - no value binding
+ * - no context variables
+ * - zone agnostic
+ * - suspense template
+ *
+ * @docsCategory ChunkDirective
+ * @docsPage ChunkDirective
+ * @publicApi
+ */
+@Directive({ selector: '[rxChunk]', standalone: true })
+export class RxChunk implements OnInit, OnDestroy {
+  #strategyProvider = inject<RxStrategyProvider<string>>(RxStrategyProvider);
+
+  /**
+   * @description
+   * The rendering strategy with which the chunk directive should schedule the view
+   * creation. If no strategy is defined, `*rxChunk` will use the configured
+   * `primaryStrategy` as fallback instead.
+   *
+   * @example
+   * \@Component({
+   *   selector: 'app-root',
+   *   template: `
+   *     <div *rxChunk="strategy">
+   *       chunked template
+   *     </div>
+   *   `
+   * })
+   * export class AppComponent {
+   *   strategy = 'low';
+   * }
+   *
+   * @param { RxStrategyNames<string> } strategy
+   */
+  @Input('rxChunk') strategy = this.#strategyProvider.primaryStrategy;
+
+  /**
+   * @description
+   * Setting the `patchZone` to a falsy value will cause the `*rxChunk` directive
+   * to create the `EmbeddedView` outside of `NgZone`.
+   *
+   * @example
+   * \@Component({
+   *   selector: 'app-root',
+   *   template: `
+   *     <div *rxChunk="patchZone: false">
+   *       chunked template out of NgZone
+   *     </div>
+   *   `
+   * })
+   * export class AppComponent { }
+   *
+   * @param { boolean } patchZone
+   */
+  @Input('rxChunkPatchZone') patchZone =
```

Comment:
Do we still want this one to be at input level, instead of having it at the provider level?

Reply:
well, it's anyway on provider level. I guess we can get rid of it

Reply:
we could get rid of it, yeah